### PR TITLE
A few suggestions on the models chapter.

### PR DESCRIPTION
**Pull request by arokem**
Following up on #14, here are a few suggestions. There are some changes here that are rather mild, but the main one is a suggestion to change the B0 model to be a bit more "model-like". See if it makes sense.

### CODE REVIEW

**Comment by arokem** [comment]
Sorry - looking at the models now I see that they all take S0 as an input on initialization. I think that I understand the logic behind that. I'll go ahead and revert my suggested changes to the b0 model.

File: docs/head-motion/models.md [hunk 1]
```diff
@@ -183,7 +185,7 @@ data_train, data_test = dmri_dataset.logo_split(88, with_b0=True)
 ```
 
 ### The model factory
-To permit flexibly select models, the `eddymotion` package offers a `ModelFactory` that implements the *facade design pattern*.
+To permit flexibly in selecting models, the `eddymotion` package offers a `ModelFactory` that implements the *facade design pattern*.
```
Comment (oesteban) [inline docs/head-motion/models.md:188]:
```suggestion
To permit flexibility in selecting models, the `eddymotion` package offers a `ModelFactory` that implements the *facade design pattern*.
```

File: docs/head-motion/models.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ kernelspec:
...
+Inference of the reference map is achieved by first fitting some diffusion model (which we will draw from [DIPY](https://dipy.org)) using all data, except the particular DW map that is to be aligned. We call this scheme "leave one gradient out" or "logo".
 
 All models are required to offer the same API (application programmer interface):
 
-1. The initialization takes a gradient table as first argument, and then arbitrary parameters as keyword arguments.
-2. A `fit(data)` method, which only requires a positional argument `data`.
-3. A `predict(gradient)` method, which only requires a `gradient` entry (b-vector and b-value).
+1. The initialization takes a DIPY `GradientTable` as the first argument, and then arbitrary parameters as keyword arguments.
+2. A `fit(data)` method, which only requires a positional argument `data`, a 4D array with DWI data.
+3. A `predict(gradient_table)` method, which only requires a `GradientTable` as input. This method produces a prediction of the signal for every voxel in every direction represented in the input `gradient_table`.
```
Comment (oesteban) [inline docs/head-motion/models.md:22]:
```suggestion
3. A `predict(gradient_table)` method, which only requires a `GradientTable` as input.
   This method produces a prediction of the signal for every voxel in every direction represented in the input `gradient_table`.
```

File: docs/head-motion/models.md [hunk 3]
```diff
@@ -13,13 +13,13 @@ kernelspec:
 # Diffusion modeling
 
 The proposed method requires inferring a motion-less, reference DW map for a given diffusion orientation for which we want to estimate the misalignment.
-Inference of the reference map is achieved by first fitting some diffusion model (which we will draw from [DIPY](https://dipy.org)) using all data, except the particular DW map that is to be aligned.
+Inference of the reference map is achieved by first fitting some diffusion model (which we will draw from [DIPY](https://dipy.org)) using all data, except the particular DW map that is to be aligned. We call this scheme "leave one gradient out" or "logo".
```
Comment (oesteban) [inline docs/head-motion/models.md:16]:
```suggestion
Inference of the reference map is achieved by first fitting some diffusion model (which we will draw from [DIPY](https://dipy.org)) using all data, except the particular DW map that is to be aligned.
We call this scheme "leave one gradient out" or "logo".
```

**Review by oesteban — APPROVED** [review]
Three minor suggestions

**Comment by arokem** [comment]
Thanks for the added suggestions! 🙌

**Pull request merged**